### PR TITLE
feat: article card

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://app.clickup.com/t/8678v0j09, to test run storybook pnpm storybook and go to articleCard

The collections and article data are hardcoded, probably will evolve but think is a good starting point

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/js/Components/Articles/ArticleCard/ArticleCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    alt="TBD"
+                    src="https://images.unsplash.com/photo-1638803040283-7a5ffd48dad5?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2592&q=80"
+                />
+            </div>
+
+            <div className="flex flex-1 flex-col px-6 py-3">
+                <div className="text-sm font-medium text-theme-secondary-700">24 Oct 2023</div>
+
+                <h4 className="mt-1 line-clamp-2 text-lg font-medium leading-7 text-theme-secondary-900">
+                    Japan Eyeing a Digital Transformation With NFTs and more text that should be truncated
```
Comment (ItsANameToo) [inline resources/js/Components/Articles/ArticleCard/ArticleCard.tsx:54]: image

this still seems to show the text that gets truncated (the black "dots") on the line below the visible text

File: stories/Articles/ArticleCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        article,
+    }: {
+        collections: Array<Pick<App.Data.Nfts.NftCollectionData, "image" | "slug">>;
+        article: any;
+    }) => {
+        return (
+            <ArticleCard
+                collections={collections}
+                article={article}
+            />
```
Comment (ItsANameToo) [inline stories/Articles/ArticleCard.stories.tsx:20]: add a few more variants, e.g.:
- 1 featured collection
- 3 featured collections
- 8 featured collections
- short title that is not truncated
- long title that is truncated
- add a grid overview where you show 4-5 cards next to each other to highlight responsiveness

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/js/Components/Articles/ArticleCard/ArticleCard.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                </span>
+
+                <div className="flex">
+                    {collections.map((collection) => (
+                        <ArticleCardCollection
+                            key={collection.slug}
+                            collection={collection}
+                        />
+                    ))}
+                </div>
```
Comment (ItsANameToo) [inline resources/js/Components/Articles/ArticleCard/ArticleCard.tsx:70]: this requires https://app.clickup.com/t/8678v9pvd 🤔

File: resources/js/Components/Articles/ArticleCard/ArticleCard.tsx (hunk 1 above)
Comment (alfonsobries) [reply resources/js/Components/Articles/ArticleCard/ArticleCard.tsx:54]: @ItsANameToo i cant replicate that 🤔 , some cache issue or smomething like that maybe?  can you try again?

File: stories/Articles/ArticleCard.stories.tsx (hunk 2 above)
Comment (alfonsobries) [reply stories/Articles/ArticleCard.stories.tsx]: except for the 8 features collections that require a component rest is ready

File: resources/js/Components/Articles/ArticleCard/ArticleCard.tsx (hunk 3 above)
Comment (alfonsobries) [reply resources/js/Components/Articles/ArticleCard/ArticleCard.tsx:70]: @ItsANameToo we can replace once ready

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**